plugins/cue_points/quiz/services/QuizService.php

```diff
@@ -58,7 +58,7 @@ public function updateAction( $entryId, KalturaQuiz $quiz )
 	{
 		$dbEntry = entryPeer::retrieveByPK($entryId);
 		$kQuiz = QuizPlugin::validateAndGetQuiz( $dbEntry );
```

Comment:
Instead of fetching the kQuiz here and passing it into validateAndUpdateQuizData, just fetch it in validateAndUpdateQuizData, it's not used here anyway, you also won't need to pass the version.